superset/views/core.py

```diff
@@ -2650,7 +2650,12 @@ def validate_sql_json(self):
                 return json_error_response(f"{msg}")
 
     def _sql_json_async(
-        self, session: Session, rendered_query: str, query: Query, expand_data: bool
+        self,
+        session: Session,
+        rendered_query: str,
+        query: Query,
+        expand_data: bool,
+        log_params: dict,
```

Comment:
I'm guessing these are more specifically along the lines of `Dict[str, str]`?

Reply:
I'll put Dict[str, Any] to allow flexibility for future logs.